### ISSUE TITLE
Can't install on Debian. Can't draw. Can't edit in external soft. No information at start wait. No shot-generator.

### ISSUE THREAD

**Issue by quirinux-so** [wonderunit/storyboarder#1685]
Greetings from Argentina, I don't speak English well.

I am the developer of Quirinux, a linux distribution made for animated cinema. I have several problems with Storyboarder.

_Storyboarder 1.10.1
Debian 9.9
i5 - 8 gb ram
intel hd graphics_

**Install problems**

When I try to run the appimage in Debian, the following appears:

> ./Storyboarder-1.10.1-linux-x86_64.AppImage 
> [2601:0620/111509.248908:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /tmp/.mount_Storyb8scxQJ/chrome-sandbox is owned by root and has mode 4755.
> `trap' para punto de parada/seguimiento

**Draw serious problems**

On the other hand, although I can run it with the flag --no-sandbox at the end, I wanted to tell you that I still have problems that are dragging on from previous versions: specifically a very poor performing, even with all preference options disabled. Drawing is impossible for me, above all, because when you draw you get stuck, it's as if when you put the pencil down it start a process and when you lift it it stop. That causes constant jumps, badly made lines, it is impossible to draw like that. I clarify that it doesn't happen to me with Krita, Gimp, Opentoonz or MyPaint. 

**Edit in external soft problem**

The function to edit in Photoshop (in my case GIMP) does not open the files, only the GIMP. And if I try to assign Krita, as I intend to use the latest version, in Debian I only have the option of flatpak, snap or appimage. None of the three is recognized by Storyboarder. 

**No Information at start wait time:** 

Since the program takes so long (several minutes) to start, try to put a splash in the meantime, because the average user thinks that the program is not starting and end up clicking several times, with the result that several instances are started. It is important to give this process information to the user. 

**No shot generator**

The shot generator window shows absolutely nothing.

**Suggestion about updates:**

It would be good to be able to disable, from preferences, the check for updates at startup. And it would be great also to be able to download old versions, since at this moment I would like to be able to go back to 1.8 or 1.7 and see if then I can draw. 

----------------------------------

_The truth is that the idea of the program is wonderful, but I could never use it. The most serious thing is not being able to draw._

**Comment by asilnevs** [comment]
It's still possible to download the old versions from here: https://github.com/wonderunit/storyboarder/releases/

**Comment by setpixel** [comment]
Can you try the latest version?

**Comment by quirinux-so** [comment]
> Can you try the latest version?

Hello! 

Version 1.12.0 on Debian 9 Stretch and Debian 10 Buster, not work. Returns:

./Storyboarder-1.12.0-linux-x86_64.AppImage 
[5885:0628/163713.390523:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /tmp/.mount_StorybkK9WBw/chrome-sandbox is owned by root and has mode 4755.

**Comment by quirinux-so** [comment]
> It's still possible to download the old versions from here: https://github.com/wonderunit/storyboarder/releases/

Thank you. [Chuckles] The link you gave me served to see if previous versions can be installed. And yes, they can, but they work very slowly on my i5 with 8 GB ram. I don't understand why this software is so heavy, being that I can handle others like Blender, Kdenlive, Krita, qStopmotion and OpenToonz with total fluidity.

**Comment by audionerd** [comment]
Storyboarder is built using Electron, which has an open issue that I think you're running into. Here's a thread with more info on how to patch it: https://github.com/electron/electron/issues/17972
Have you tried the Snap package instead?

**Comment by quirinux-so** [comment]
Finally, I repackaged Storyboarder in a Debian Buster compatible .deb installer. Works. I share it.

http://my.opendesktop.org/s/GePzTyxoYpM622t/download

**Comment by audionerd** [comment]
Is there anything you’d recommend we change about the build/packaging system for the next official release?

**Comment by quirinux-so** [comment]
> Is there anything you’d recommend we change about the build/packaging system for the next official release?

My suggestion would be to review .appimage can work for Debian out the box. 

I couldn't tell you why, but this is what happens when I try to run the official .appimage in a fresh Debian 10 Buster installation. 

<img src="https://my.opendesktop.org/apps/files_sharing/publicpreview/7a6aj2NgKHDxt7T?x=1366&y=265&a=true&file=story-issue.png&scalingup=0" alt="screenshot">

The program only works for me by extracting the .appimage, so my solution was to build the .deb which also speeds up the start. The other option I tried successfully was to start the .appimage with  --nosandbox flag and it works too, although it takes a little longer to start.

I have not tried the snap version, in general I did not have a good experience with that system so I don't have it installed. One of the reasons I use Debian is because I am not very happy with the excessive snap implementation in Ubuntu. Regards :-)
